source/class/qxl/apiviewer/ui/panels/MethodPanel.js

```diff
@@ -33,7 +33,7 @@ qx.Class.define("qxl.apiviewer.ui.panels.MethodPanel", {
      * @Override
      */
     canDisplayItem(dao) {
-      return dao instanceof qxl.apiviewer.dao.Method && !dao.isStatic();
+      return dao instanceof qxl.apiviewer.dao.Method && !dao.isStatic() || dao instanceof qxl.apiviewer.dao.PropertyMethod;
```

Comment:
how about adding brackets for clarity

Reply:
Imo it is not hard snippet of code that brackets are needed but nevertheless:
1. `(dao instanceof qxl.apiviewer.dao.Method && !dao.isStatic()) || dao instanceof qxl.apiviewer.dao.PropertyMethod`
2. `(dao instanceof qxl.apiviewer.dao.Method || dao instanceof qxl.apiviewer.dao.PropertyMethod) && !dao.isStatic()`

It is probably imposibble 2 case but in theory may happens.